src/models/Offer/Model.ts

```diff
@@ -4,13 +4,20 @@ import {
   Model,
   Table,
   ForeignKey,
-  BelongsTo
+  BelongsTo, Is
 } from "sequelize-typescript";
 import { HasOneGetAssociationMixin } from "sequelize";
-
 import { Company } from "../Company";
+import { validatePositiveNumber, validateSalaryRange } from "validations-fiuba-laboral-v2";
 
-@Table
+@Table({
+  tableName: "Offers",
+  validate: {
+    validateSalaryRange(this: Offer) {
```

Comment:
salaryRange: offer => validateSalaryRange(this.minimumSalary, this.maximumSalary);

esto creo que deberia funcionar y es mas conciso

(tal vez haga falta tipar el Offer igual, supongo que tan inteligente no es)